javascript/10-Regular-Expression-Matching.js

```diff
@@ -0,0 +1,8 @@
+/**
+ * @param {string} s
+ * @param {string} p
+ * @return {boolean}
+ */
+var isMatch = function(s, p) {
+    return eval("/^"+ p + "$/").test(s);

```

Comment:
I really like the ingenuity but I don't know if I can merge this 😄

Reply:
Here is a different solution.

* **File(s) Modified**: 10-Regular-Expression-Matching.js
* **Language(s) Used**: javascript
* **Submission URL**: https://leetcode.com/submissions/detail/765132489/